addons/website_forum/tests/test_forum_tag.py

```diff
@@ -0,0 +1,35 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
```

Comment:
missing `# -*- coding: utf-8 -*-` before that line

Reply:
Done. I thought this got deprecated though, but maybe only in later versions? Never got the "official memo".

Reply:
Is it ? they are way more file with the encoding than without

Reply:
I don't think we'd go as far as removing them on existing files.